dnsimple/struct/batch_change_zone_records.py

```diff
@@ -0,0 +1,98 @@
+import json
+from dataclasses import dataclass
+
+import omitempty
+
+from dnsimple.struct import Struct
+from dnsimple.struct.zone_record import ZoneRecord
+
+
+@dataclass
+class BatchChangeZoneRecordsUpdateInput(dict):
+    """Represents a zone record update input for a batch operation"""
+
+    def __init__(self, id, name=None, content=None, ttl=None, priority=None, regions=None):
+        dict.__init__(self, id=id, name=name, content=content, ttl=ttl, priority=priority, regions=regions)
+
+    def to_json(self):
+        omitted = omitempty(self)
+
+        if self['name'] == '':
+            omitted['name'] = ''
+
+        return json.dumps(omitted)
+
+
+@dataclass
+class BatchChangeZoneRecordsDeleteInput(dict):
+    """Represents a zone record deletion input for a batch operation"""
+
+    def __init__(self, id):
+        dict.__init__(self, id=id)
+
+
+@dataclass
+class BatchChangeZoneRecordsInput(dict):
+    """Represents the data to send to the DNSimple API to make a batch change on the records of a zone
+
+    All parameters are optional - you can perform creates only, updates only, deletes only,
+    or any combination of the three operations.
+
+    :param creates: List[ZoneRecordInput] - Records to create (optional)
```

Comment:
I wonder if there is a way for us to alias this class `ZoneRecordInput` as `BatchChangeZoneRecordsCreateInput` for consistency and clarity.

Reply:
Great! Thanks for doing that 👏